satellite/satellitedb/satellitedbtest/run.go

```diff
@@ -70,6 +70,30 @@ func SchemaName(testname, category string, index int, schemaSuffix string) strin
 	return strings.ToLower(testname + "/" + schemaSuffix + "/" + category + indexStr)
 }
 
+// CreateMasterDB creates a new satellite database for testing
+func CreateMasterDB(t *testing.T, category string, index int, dbInfo Database) (db satellite.DB, err error) {
+	if dbInfo.URL == "" {
+		t.Fatalf("Database %s connection string not provided. %s", dbInfo.Name, dbInfo.Message)
+	}
+
+	schemaSuffix := SchemaSuffix()
+	t.Log("schema-suffix ", schemaSuffix)
+
+	log := zaptest.NewLogger(t)
+	schema := SchemaName(t.Name(), category, index, schemaSuffix)
+
+	switch dbInfo.Name {
+	case "Postgres":
+		db, err = NewPostgres(log.Named("db"), schema)
+	case "Cockroach":
+		namespace := strings.ReplaceAll(schema, "/", "")
```

Comment:
probably need to take out this ReplaceAll too

Reply:
oh missed that one, let me remove